pkg/resourceinterpreter/customizedinterpreter/customized.go

```diff
@@ -32,8 +33,113 @@ type CustomizedInterpreter struct {
 	clientManager *webhookutil.ClientManager
 }
 
+// GetReplicas returns the desired replicas of the object as well as the requirements of each replica.
+// return matched value to indicate whether there is a matching hook.
+func (e *CustomizedInterpreter) GetReplicas(object *unstructured.Unstructured) (replica int32, requires *workv1alpha2.ReplicaRequirements, enabled bool, err error) {
```

Comment:
Why move these functions ahead of `NewCustomizedInterpreter()`? It becomes hard to see the differences.